classes/person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :age, :name
-  attr_reader :id, :parent_permission
+  attr_reader :id, :parent_permission, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = generate_id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] As stated in the project requirement when you set up many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, A person should have many rentals:  So there should be `method for adding rentals` just like the below sample , Kindly Adjust it.

```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```